harvester/records/formats/marc.py

```diff
@@ -242,3 +374,30 @@ def _bbox_convert_coordinate_string_to_decimal(
         getcontext().prec = original_precision
 
         return decimal_value
+
+    def get_date_date_strings(self) -> list[str]:
```

Comment:
`get_date_date_strings` Is this supposed to be repeated? Find-replace error perhaps?

Reply:
Ack!  Ha!  Yeah, unsure.  Will fix, thanks!